docs/devices/TH1123ZB.md

```diff
@@ -49,15 +56,30 @@ If you want to automate the publishing of the outdoor temperature using Home Ass
     service: mqtt.publish
 ```
 
-### Enabling time
-To enable _time_ you need to send a _blank_ message to the following MQTT topic:
+### Activate eco mode
```

Comment:
Please don't modify below `<!-- Notes END: Do not edit below this line -->` add this to the note section.